druid/src/test/java/org/apache/calcite/test/DruidAdapter2IT.java

```diff
@@ -2902,6 +2903,7 @@ private void testCountWithApproxDistinct(boolean approx, String sql,
         .returnsUnordered("EXPR$0=86829");
   }
 
+  @Disabled("CALCITE-7271")
```

Comment:
Instead of disabling, I would simply update the plan since this can be seen as bug fix.

Reply:
Is this test still affected by the changes in current PR?